src/lib/i18n/locales/fr-FR/translation.json

```diff
@@ -201,10 +201,10 @@
 	"Description": "Description",
 	"Didn't fully follow instructions": "N'a pas entièrement respecté les instructions",
 	"Disabled": "Désactivé",
-	"Discover a function": "Découvrez une fonction",
-	"Discover a model": "Découvrir un modèle",
-	"Discover a prompt": "Découvrir une suggestion",
-	"Discover a tool": "Découvrez un outil",
+	"Discover a function": "Trouvez une fonction",
+	"Discover a model": "Trouvez un modèle",
+	"Discover a prompt": "Trouvez un prompt",
+	"Discover a tool": "Trouvez un outil",
```

Comment:
In sentences such as "Discover a model!", "Trouvez" ("Find") is a bit more natural than "Découvrez" in French.